tools/deb/test/run_iptables_test.sh

```diff
@@ -6,11 +6,12 @@
 # Script requires a working docker on the test machine
 # It is run in the proxy dir, will create a docker image with proxy deb installed
 
```

Comment:
should this also run the iptable test found along the original script (in the pilot repo)